src/gar.lua

```diff
@@ -1076,22 +1077,14 @@ function gar.pruneGateways(currentTimestamp, msgId)
 							table.insert(result.prunedGateways, gatewayAddress)
 						else
 							-- find the next prune timestamp
-							minNextEndTimestamp =
-								--- @diagnostic disable-next-line: param-type-mismatch
-								math.min(minNextEndTimestamp or gateway.endTimestamp, gateway.endTimestamp)
+							gar.scheduleNextGatewaysPruning(minNextEndTimestamp)
```

Comment:
This local `minNextEndTimestamp ` was removed up top, right? And isn't the thing that needs to be considered for scheduling gateway.endTimestamp?

Reply:
good catch, this needs to be updated wherever we were using `minNextEndTimestamp`

Reply:
[f4d8ec2](https://github.com/ar-io/ar-io-network-process/pull/347/commits/f4d8ec2dbfe972e41d77cbc61c8072dae7302ebd)